### ISSUE TITLE
Accessibility bug: Screen reader does not announce the updated results in the output section after activating "Run" button.

### ISSUE THREAD

**Issue by LadyNaggaga** [MicrosoftDocs/feedback#2530]
You can see a video of the bug [here](https://github.com/dotnet/try/files/4207879/788_A11y_.NETOnlineREPL_.NET.Editor_Status.Message.pptx)
**Environment details:**
https://docs.microsoft.com/
Browser: Chromium Edge Version 80.0.361.50 (Official build) (64-bit)
OS Details:Windows 10 

#A11yTCS;#A11ySev2;#A11y_.NETOnlineREPL;#A11yWCAG2.1;#.NET Online REPL;#Benchmark;#DesktopWeb;#Win10;#WCAG4.1.3;#Accessibility;#.NET Online REPL;#ChromiumEdge;

**​Repro Steps:​**
1.Hit the URL "https://docs.microsoft.com/" and login with appropriate credentials.
2.Tab till "Enter focus Mode" button and press enter.
3.Start Narrator and tab till Run button and hit enter.
4.Observe the narrator reading.

**Actual Results:​**
Screen reader does not announce the updated status in the output section after activating "Run" button.
After activating the Run button the status gets updated in the output section which is not announced by the screen reader.

**Same issue with Chrome + NVDA .**

**Expected Result:​**
After activating the Run button the status gets updated in the output section which must be announced by the screen reader.

**User Impact:​**
Screen readers users will not be aware about the results updated in the output section.

**Attachment:**
[788_A11y_.NETOnlineREPL_.NET Editor_Status Message.pptx](https://github.com/dotnet/try/files/4207879/788_A11y_.NETOnlineREPL_.NET.Editor_Status.Message.pptx)

**Comment by AbhitejJohn** [comment]
From the comments [here](https://github.com/dotnet/try/issues/788) this can be closed out.
